helpers/localizePrice.js

```diff
@@ -0,0 +1,38 @@
+'use strict';
+const common = require('./lib/common.js');
+
+const factory = globals => {
+    return function(price, locale) {
```

Comment:
Does the locale value (coming from Stencil context) takes into consideration of the locale support by the theme? i.e.: The browser could forward an `accept-language` header that the theme could not fulfil. In that case, the price would be localized according to the locale value but not the rest of the page. But I guess it's not really a big issue (as it's understandable by the shopper)?

Reply:
No, it wouldn't take that into account; it's a straight representation of our parsed result of the browser's `Accept-Language` header.

That being said, I don't think that's a bad thing - localizing price display is a different task than localizing all human-readable language, and perhaps we can consider those tasks separately?